applications/nrf5340_audio/src/audio/audio_system.c

```diff
@@ -90,7 +91,7 @@ static void audio_headset_configure(void)
 	}
 #endif /* (CONFIG_STREAM_BIDIRECTIONAL) */
 
-	sw_codec_cfg.decoder.num_ch = SW_CODEC_MONO;
+	sw_codec_cfg.decoder.num_ch = IS_ENABLED(CONFIG_LC3_PLAYBACK) ? 2 : SW_CODEC_MONO;
```

Comment:
Would be great if we can add a comment in here to explain that LC3 playback module also need a decoder, so the number of channel for decoder is 2.